packages/neo-one-node-bin/src/utils/setupNode.ts

```diff
@@ -0,0 +1,107 @@
+// tslint:disable no-object-mutation no-any
```

Comment:
This should probably just live in src (i.e. no reason to have a utils directory at all)